rainfall/style.css

```diff
@@ -15,11 +15,6 @@ Text Domain: rainfall
 Tags: one-column, custom-colors, custom-menu, custom-logo, editor-style, featured-images, full-site-editing, rtl-language-support, theme-options, threaded-comments, translation-ready, wide-blocks
 */
 
-.wp-block-post-title:not(.editor-post-title) {
```

Comment:
This is a cool effect but it is too generic and the selector is too generic and will break anytime a user inserts a post title, without them being able to customize it.

Reply:
I pushed a commit to do this in a different way

Reply:
Thanks, this works well!